src/core/utils/assembly.ts

```diff
@@ -19,19 +19,20 @@ export interface ChromSize {
 /**
  * Get relative chromosome position (e.g., `100` => `chr:100`)
  */
-export function getRelativeGenomicPosition(absPos: number, assembly?: string): string {
+export function getRelativeGenomicPosition(absPos: number, assembly?: string): GenomicPosition {
     const chrAndRange = Object.entries(GET_CHROM_SIZES(assembly).interval).find(d => {
```

Comment:
Maybe destructure here, rather than below:

```javascript
    const [chromosome, absInterval] = Object.entries(GET_CHROM_SIZES(assembly).interval).find(d => {
```

Reply:
I like this way